app/helpers/is-modified-longer-then-6-months.js

```diff
@@ -8,9 +8,7 @@ export function isModifiedWithinLast6Months([modifiedDate]) {
   let result = {};
   if (format(today, 'dd-MM-yyyy') === format(modifiedDate, 'dd-MM-yyyy')) {
     result = { skin: 'success', icon: 'circle-check' };
-  } else if (isBefore(new Date(modifiedDate), sixMonthsAgo)) {
-    result = { skin: 'warning', icon: 'alert-triangle' };
-  } else if (format(modifiedDate, 'dd-MM-yyyy') === undefined || null) {
+  } else if (isBefore(new Date(modifiedDate), sixMonthsAgo) || !modifiedDate) {
```

Comment:
By itself this change does not prevent the error from being thrown. It is the call to `format(modifiedDate, 'dd-MM-yyyy')` on line 9 that throws an error when `modifiedDate` is `undefined`. (Which is why I suggested the extra condition there previously.)